NPE crash in util.ConfigXml.getFolders (ConfigXml.java:486) [gplay report]

```
13. Juli 14:53 in der App-Version 1020005
lavender, Android 9
Bericht 1

java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3722)
  at android.app.ActivityThread.access$1700 (ActivityThread.java:200)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1704)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:201)
  at android.app.ActivityThread.main (ActivityThread.java:6810)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:547)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:873)
Caused by: java.lang.NullPointerException: 
  at com.nutomic.syncthingandroid.util.ConfigXml.getAttributeOrDefault (ConfigXml.java:396)
  at com.nutomic.syncthingandroid.util.ConfigXml.getFolders (ConfigXml.java:486)
  at com.nutomic.syncthingandroid.service.SyncthingService.applyCustomRunConditions (SyncthingService.java:435)
  at com.nutomic.syncthingandroid.service.SyncthingService.lambda$UodbKdkde829Ytft6f0hO77Hrkc (Unknown Source)
  at com.nutomic.syncthingandroid.service.-$$Lambda$SyncthingService$UodbKdkde829Ytft6f0hO77Hrkc.onSyncPreconditionChanged (Unknown Source:2)
  at com.nutomic.syncthingandroid.service.RunConditionMonitor.updateShouldRunDecision (RunConditionMonitor.java:268)
  at com.nutomic.syncthingandroid.service.RunConditionMonitor.<init> (RunConditionMonitor.java:166)
  at com.nutomic.syncthingandroid.service.SyncthingService.onStartCommand (SyncthingService.java:284)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3700)
```

--- a/app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java
+++ b/app/src/main/java/com/nutomic/syncthingandroid/util/ConfigXml.java
@@ -483,20 +483,22 @@ public class ConfigXml {
             */
             folder.versioning = new Folder.Versioning();
             Element elementVersioning = (Element) r.getElementsByTagName("versioning").item(0);
-            folder.versioning.type = getAttributeOrDefault(elementVersioning, "type", "");
-            NodeList nodeVersioningParam = elementVersioning.getElementsByTagName("param");
-            for (int j = 0; j < nodeVersioningParam.getLength(); j++) {
-                Element elementVersioningParam = (Element) nodeVersioningParam.item(j);
-                folder.versioning.params.put(
-                        getAttributeOrDefault(elementVersioningParam, "key", ""),
-                        getAttributeOrDefault(elementVersioningParam, "val", "")
-                );
-                /*
-                Log.v(TAG, "folder.versioning.type=" + folder.versioning.type +
-                        ", key=" + getAttributeOrDefault(elementVersioningParam, "key", "") +
-                        ", val=" + getAttributeOrDefault(elementVersioningParam, "val", "")
-                );
-                */
+            if (elementVersioning != null) {
+                folder.versioning.type = getAttributeOrDefault(elementVersioning, "type", "");
+                NodeList nodeVersioningParam = elementVersioning.getElementsByTagName("param");
+                for (int j = 0; j < nodeVersioningParam.getLength(); j++) {
+                    Element elementVersioningParam = (Element) nodeVersioningParam.item(j);
+                    folder.versioning.params.put(
+                            getAttributeOrDefault(elementVersioningParam, "key", ""),
+                            getAttributeOrDefault(elementVersioningParam, "val", "")
+                    );
+                    /*
+                    Log.v(TAG, "folder.versioning.type=" + folder.versioning.type +
+                            ", key=" + getAttributeOrDefault(elementVersioningParam, "key", "") +
+                            ", val=" + getAttributeOrDefault(elementVersioningParam, "val", "")
+                    );
+                    */
+                }
             }
 
             // For testing purposes only.

